packages/calcite-design-tokens/support/token-transformer/utils/regex.ts

```diff
@@ -0,0 +1,9 @@
+// Match files and tokens which include "[" or "]"
+export const matchPlaceholderElement = /[\[\]]/g;
+
+// Match custom variable indicator created by Designers in Token Studio.
+// Using the ${token.name} format in Token Studio makes it much easier for Designers to search for token references while doing their work.
+export const tokenStudioCustomVariableIndicator = /\$[.\w-]+/g;
+
+// Match files and tokens which include "backup", "[", "]", or file extensions
+export const matchExclusions = /(backup|\[|\])(?=\.\w+$)/;
```

Comment:
Is this still necessary now that we removed the backup file and the file with brackets? Same with `matchPlaceholderElement`